test/utils/common.ts

```diff
@@ -25,7 +28,8 @@ export async function tearDown() {
 
 export async function getNewPage() {
   let page: Page;
-  if (process.env.NODE_ENV === 'development') {
+  const testingLocalhost = testUrl.indexOf('localhost') !== -1;
+  if (testingLocalhost) {
```

Comment:
Perhaps reusing the const in line 10:
```suggestion
  if (localhost) {
```